src/components/CopyToClipboard/CopyToClipboard.ts

```diff
@@ -87,6 +81,11 @@ export class CopyToClipboard extends ResultAction {
             this.copyToClipboardFallback(text);
         }
         this.setToolipText(this.options.successTooltip);
+        this.refreshTooltip();
+    }
+
+    private refreshTooltip() {
+        setTimeout(() => this.setToolipText(this.options.tooltip), 500);
```

Comment:
Can you explain why this is required now ? What was the issue with the mouseleave ?

Reply:
Yeah the issue with mouse leave is that the event is not consistent enough.
For instance if I pass my mouse through the tooltip no mouseleave will be trigger on the parent element.